modules/button/react/README.md

```diff
@@ -825,3 +828,51 @@ Default: `undefined`
 > The icon of the button. Optional because ToolbarIconButton can also wrap a SystemIcon component.
 
 ---
+
+# ToolbarDropdownButton
+
+> Button containing an icon or custom element. Icon may be a component from
```

Comment:
Can you add a note here and for `ToolbarIconButton` about their specific usage for toolbars?